shuffle/cli/Cargo.toml

```diff
@@ -12,6 +12,9 @@ edition = "2018"
 [dependencies]
 anyhow = "1.0.38"
 bcs = "0.1.2"
+chrono = "0.4"
+chrono-tz = "0.6.0"
```

Comment:
As mentioned earlier, let's remove this dependency and use the std library for second from EPOCH